ACHNBrowserUI/ACHNBrowserUI/views/todayDashboard/TodayChoresSection.swift

```diff
@@ -24,7 +24,7 @@ struct TodayChoresSection: View {
 
     var body: some View {
         Section(header: SectionHeaderView(text: "Chores", icon: "checkmark.seal.fill")) {
-            NavigationLink(destination: ChoreListView()) {
+            NavigationLink(destination: LazyView(ChoreListView())) {
```

Comment:
As of Xcode 11.4.1, iOS 13.4.1, Apple fixed NavigationLink destination loading and views are now loaded lazily. So this shouldn’t be necessary?

Reply:
I tried without, but I still face the problem.

**Without Lazy**: When hiding Finished Task and adding new chores, it resets and shows all the chores.

![wo](https://user-images.githubusercontent.com/139272/83949334-6f215a80-a823-11ea-9249-d890c16dd0a5.gif)

**With Lazy**: When hiding Finished Tasks and adding new chores, it behaves as expected.

![w](https://user-images.githubusercontent.com/139272/83949358-9e37cc00-a823-11ea-9852-d6a8e3bbb19f.gif)

Reply:
Interesting 🤔